tx/textual/internal/testpb/1.proto

```diff
@@ -55,3 +55,36 @@ message Bar {
   string bar_id = 1;
   bytes  data   = 2;
 }
+
+// Baz is a sample message type used for testing repeated rendering.
+message Baz {
+  repeated Foo    messages                       = 1;
+  repeated string string_messages                = 2;
+  Ballot          vote                           = 3;
+  repeated cosmos.base.v1beta1.Coin price        = 4;
+  repeated google.protobuf.Timestamp expirations = 5;
```

Comment:
we should add a repeated Enum too. But let's leave it like this for now, maybe we merge this first and I'll do it in #13853.